can-attribute-encoder-test.js

```diff
@@ -10,6 +10,8 @@ QUnit.test('encoding / decoding', function() {
 		'on:fooBar:by:bazQuz': 'on:foo:u:bar:by:baz:u:quz',
 		'vm:sProp:to': 'vm:s:u:prop:to',
 		'fooBar:to': 'foo:u:bar:to',
+		'FooBar:to': ':u:foo:u:bar:to',
+		'Foobar:to': ':u:foobar:to',
```

Comment:
Can you add these cases to the `encoded values should work with setAttribute` test also? I know the spec says attributes can start with `:`, but it would be good to have our tests confirm it.

Reply:
done!